app/src/main/java/ai/elimu/appstore/synchronization/AppListAdapter.java

```diff
@@ -0,0 +1,262 @@
+package ai.elimu.appstore.synchronization;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Environment;
+import android.support.v4.content.FileProvider;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import java.io.File;
+import java.util.List;
+
+import ai.elimu.appstore.BaseApplication;
+import ai.elimu.appstore.BuildConfig;
+import ai.elimu.appstore.R;
+import ai.elimu.appstore.dao.ApplicationVersionDao;
+import ai.elimu.appstore.model.Application;
+import ai.elimu.appstore.model.ApplicationVersion;
+import ai.elimu.appstore.util.UserPrefsHelper;
+import ai.elimu.model.enums.admin.ApplicationStatus;
+import timber.log.Timber;
+
+/**
+ * Created by "Tuan Nguyen" on 11/14/2016.
```

Comment:
Don't use "Created by ...". Instead, leave it blank. This makes future less developers to refactor the code since the code ownership will be of the entire team and not one single developer.